vpc.tf

```
@@ -0,0 +1,109 @@
+# VPC
+resource "aws_vpc" "task1_vpc" {
+  cidr_block       = var.vpc_cidr
+  instance_tenancy = "default"
+  tags = {
+    Name = "task1_vpc"
+  }
```

Please separate the `tags` block from the rest of arguments here and everywhere else in the code

```suggestion
  cidr_block       = var.vpc_cidr
  instance_tenancy = "default"
  
  tags = {
    Name = "task1_vpc"
  }
```